specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2018-08-01-preview/managedClusters.json

```diff
@@ -1120,8 +1120,7 @@
       "properties": {
         "adminUsername": {
           "type": "string",
-          "description": "The administrator username to use for Windows VMs.",
-          "pattern": "^[a-zA-Z0-9]+([._]?[a-zA-Z0-9]+)*$"
+          "description": "The administrator username to use for Windows VMs."
```

Comment:
Suggest to update the description as that in compute model.

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json#L9443-L9450